src/arizona_socket.erl

```diff
@@ -124,77 +124,5 @@ push_event(Name, Payload, #{events := Events} = Socket) ->
 prune(Socket) ->
     Socket#{
         events => [],
-        changes => #{}
+        changes => ordsets:new()
     }.
-
-%% --------------------------------------------------------------------
-%% EUnit
-%% --------------------------------------------------------------------
-
--ifdef(TEST).
--compile([export_all, nowarn_export_all]).
--include_lib("eunit/include/eunit.hrl").
-
-render_block_test() ->
-    ?assertEqual(
-       rendered(), arizona_tpl_render:render_block(block(#{}), #{
-            title => <<"Arizona">>,
-            view_count => 0,
-            decr_btn_text => <<"Decrement">>})).
```

Comment:
Unless we're doing this for good reason, I'd not start moving EUnit to CT at this moment (especially given this PR's size, already).

Reply:
(remember we have https://github.com/arizona-framework/arizona/issues/155)

Reply:
This test is duplicated from the new renderer module, so it's irrelevant.